index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Israel CHIZUNGU Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <section>
+        <!--  toolbar starts -->
```

Comment:
- [ **OPTIONAL** ] As I mentioned above, I think it would be better if you remove this `<section>` tag(**both the opened one and the closed one**) for good semantic tags. You can see an example below:
```HTML
  <header>
    code of the header here...
  </header>

  <main>
    <section>
      code for the 1st section...
    </section>

    <section>
      code for the 2nd section...
    </section>
    ...
  </main>

  <footer>
    code for the footer
  </footer>
```